cmd/dfget/app/root.go

```diff
@@ -145,22 +146,48 @@ func transParams() error {
 
 // initClientLog init client logger.
 func initClientLog() error {
-	var (
-		logPath  = path.Join(cfg.WorkHome, "logs")
-		logLevel = "info"
-	)
+	// set the log level
+	logLevel := logrus.InfoLevel
```

Comment:
create a function to initialize logrus, then the `dfget` and `dfget server` could reuse this function to init log.